pkgs/development/python-modules/octodns-hetzner/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, buildPythonPackage
+, certifi
+, charset-normalizer
+, dnspython
+, fetchFromGitHub
+, fqdn
+, idna
+, natsort
+, octodns
+, python-dateutil
+, pythonOlder
+, pyyaml
+, requests
+, setuptools
+, six
+, urllib3
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "octodns-hetzner";
+  version = "0.0.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "octodns";
+    repo = "octodns-hetzner";
+    rev = "v${version}";
+    hash = "sha256-dbx9t50NCWq3QdVRh5ONyBNKQ6BvbZl2vzknTSp6Q4M=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    certifi
+    charset-normalizer
+    dnspython
+    fqdn
+    idna
+    natsort
+    octodns
+    python-dateutil
+    pyyaml
+    requests
+    six
+    urllib3
+  ];
+
```

```suggestion
  env.OCTODNS_RELEASE = 1;
```
This variable may be necessary to set up the correct version.
https://github.com/octodns/octodns-hetzner/blob/6c14dd9d15d7ce282b06a608b6621faeb0211130/setup.py#L21-L24